### PR TITLE
build: post 10.1.14 changes

### PR DESCRIPTION
**Pull request by johanandren**
References #xxxx

### CODE REVIEW

File: project/MiMa.scala [hunk 1]
```diff
@@ -61,6 +61,7 @@ object MiMa extends AutoPlugin {
     "10.1.11",
     "10.1.12",
     "10.1.13",
+    "10.1.13",
```
Comment (ignasi35) [inline project/MiMa.scala:64]:
```suggestion
    "10.1.14",
```

Comment (johanandren) [reply project/MiMa.scala]: lol

**Review by ignasi35 — APPROVED** [review]

**Comment by ignasi35** [comment]
Mental note: should we enable the new GH's Automerge in Akka-HTTP?

https://github.blog/changelog/2020-12-16-pull-request-auto-merge-public-beta/

**Comment by ignasi35** [comment]
I think the callback from Jenkins got lost. Let's try to trigger it again.

**Comment by ignasi35** [comment]
OK TO TEST

**Comment by johanandren** [comment]
PLS BUILD

**Comment by jrudolph** [comment]
> Mental note: should we enable the new GH's Automerge in Akka-HTTP?

Yep, done but be careful to enable it only where safe.

**Pull request merged**